jobsets/cardano.nix

```diff
@@ -1,9 +1,11 @@
-{ pkgs ? (import <nixpkgs> {}), supportedSystems ? [ "x86_64-linux" ] }:
+let
+  fixedNixpkgs = (import <nixpkgs> {}).fetchFromGitHub (builtins.fromJSON (builtins.readFile ../nixpkgs-src.json));
+in { pkgs ? (import fixedNixpkgs {}), supportedSystems ? [ "x86_64-linux" ] }:

```

Comment:
when `jobsets/default.nix` generates a PR jobset, it will pass the wrong nixpkgs in, the one master once, not the one the PR wants

this will import the wrong one with `<nixpkgs>`, then use that to import the correct one